mono/mini/TestDriver.cs

```diff
@@ -51,6 +52,9 @@ public class TestDriver {
 				} else if (args [j] == "--exclude") {
 					exclude [args [j + 1]] = args [j + 1];
 					j += 2;
+				} else if (args [j] == "--exclude-test") {
+					exclude_test.Add (args [j + 1]);
```

Comment:
I mean, what would make the most sense I guess is if we had an --exclude, an --exclude-test, a --run-only, and a --run-only test. But I guess we can grow that fourth head if someone ever needs it :P